server/middlewares/auth.js

```diff
@@ -0,0 +1,48 @@
+const jwt = require('jsonwebtoken');
+
+const secret = process.env.SECRET || 'documentmanagement';
+
+const validation = {
+
+  /**
+   * Validate user token
+   * @method validateToken
+   * @param {object} req
+   * @param {object} res
+   * @param {function} done callback
+   * @returns {object} http callback.
+   */
+  validateToken(req, res, done) {
+    if (!req.headers.authorization) {
+      return res.status(401).send({ message: 'User unauthorised!' });
+    }
+    const jwtcode = req.headers.authorization;
+    jwt.verify(jwtcode, secret, (err, token) => (err)
+    ? res.status(401).send({ message: 'Invalid Token, User unauthorised!' })
+    : done()
+    );
+  },
+
+  /**
+   * Validate admin user
+   * @method validateAdmin
+   * @param {object} req
+   * @param {object} res
+   * @param {function} done callback
+   * @returns {object} http callback.
+   */
+  validateAdmin(req, res, done) {
```

Comment:
Can't this method leverage on `validateToken` method since they have similar contents